src/transaction.ts

```diff
@@ -253,6 +255,10 @@ export class Transaction implements TransactionStorageStructure {
       throw Error('firstRound must be a positive number');
     if (!Number.isSafeInteger(txn.lastRound) || txn.lastRound < 0)
       throw Error('lastRound must be a positive number');
+    if (txn.extraPages < 0 || txn.extraPages > 3)
+      throw Error(
+        'extra-pages must be an Integer between and including 0 to 3'
```

Comment:
nit: change this to `extraPages` because that is the name of the parameter passed in.

Reply:
done